UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -985,6 +987,34 @@ public void CmdServerReplaceItemInInventory(GameObject gameObjectSent, uint id,
 		}
 	}
 
+	[Command]
+	public void HardSuicide()
+	{
+		if (playerScript.TryGetComponent<AiPlayer>(out var aiPlayer))
+		{
+			aiPlayer.Suicide();
+			return;
+		}
+
+		if (playerScript.TryGetComponent<BlobPlayer>(out var blobPlayer))
+		{
+			blobPlayer.Death();
```

Comment:
not telling you to do it, but it would be great if we moved this stuff to interfaces like IGIbbable, so we have ISuicidable, IDeath and whatever. We would just get the interface then intstead of checking types.

Reply:
It would be even better as well if both the AI and Blob used `LivingHealthMasterBase`. but alas, they don't, because our current system did not support that. Thankfully, thanks to Bod's changes, we can move them to use that class soon and maybe even all other mobs as well.